src/fides/api/models/privacy_experience.py

```diff
@@ -219,260 +561,41 @@ class PrivacyExperience(Base):
-        ] = PrivacyExperience.get_experience_by_region_and_component(
-            db=db, region=region, component=ComponentType.privacy_center
-        )
-
-        return overlay_experience, privacy_center_experience
-
-    def unlink_experience_config(self, db: Session) -> PrivacyExperience:
-        """Removes the experience config
-
-        Note that neither the Experience Config or Experience are deleted; we're only removing
-        a FK if it exists.
-        """
-        return self.update(  # type: ignore[return-value]
-            db,
-            data={"experience_config_id": None, "experience_config_history_id": None},
-        )
-
-    def link_default_experience_config(self, db: Session) -> PrivacyExperience:
-        """Replace config from experience with default config
-
-        Replaces the FK only; does not delete Privacy Experiences or Privacy Experience Configs
-        """
-        default_config: Optional[
-            PrivacyExperienceConfig
-        ] = PrivacyExperienceConfig.get_default_config(
-            db, self.component  # type: ignore[arg-type]
-        )
-        return self.update(  # type: ignore[return-value]
-            db,
-            data={
-                "experience_config_id": default_config.id if default_config else None,
-                "experience_config_history_id": default_config.experience_config_history_id
-                if default_config
-                else None,
-            },
-        )
-
-
-def get_privacy_notices_by_region_and_component(
-    db: Session, regions: List[str], component: ComponentType
-) -> Query:
-    """
-    Return relevant privacy notices that should be displayed for a certain
-    region in a given component type
-    """
-    return (
-        db.query(PrivacyNotice)
-        .filter(PrivacyNotice.regions.overlap(regions))  # type: ignore[attr-defined]
-        .filter(
-            or_(
-                and_(
-                    component == ComponentType.overlay,
-                    PrivacyNotice.displayed_in_overlay,
-                ),
-                and_(
-                    component == ComponentType.privacy_center,
-                    PrivacyNotice.displayed_in_privacy_center,
-                ),
+                PrivacyExperienceConfig.component  # pylint: disable=comparison-with-callable
+                == component,
             )
         )
-    )
-
-
-def upsert_privacy_experiences_after_notice_update(
-    db: Session, affected_regions: List[PrivacyNoticeRegion]
-) -> List[PrivacyExperience]:
-    """
-    Keeps Privacy Experiences in sync with *PrivacyNotices* changes.
-    Create or update "overlay" or "privacy center" PrivacyExperiences based on the PrivacyNotices in the "affected_regions".
-    To be called whenever PrivacyNotices are created or updated (pass in any regions that were potentially affected)
-
-    PrivacyExperiences should not be deleted.  It's okay if no notices are associated with an Experience.
-    """
```

Comment:
Previously, both edits to a PrivacyNotice and a PrivacyExperienceConfig could cause PrivacyExperiences to be created/updated in the background to make sure we always had a valid PrivacyExperience.

Now, only ExperienceConfigs create PrivacyExperiences, when a region is added to a PrivacyExperienceConfig.